adagrams/game.py

```diff
@@ -1,11 +1,212 @@
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
+score_chart = {
+    1 : ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2 : ["D", "G"],
+    3 : ["B", "C", "M", "P"],
+    4 : ["F", "H", "V", "W", "Y"],
+    5 : ["K"],
+    8 : ["J", "X"],
+    10: ["Q", "Z"],
+}
+
+scores_chart = {
+    "A": 1, "C": 3, "B": 3, "E": 1, "D": 2, "G": 2, 
+    "F": 4, "I": 1, "H": 4, "K": 5, "J": 8, "M": 3, 
+    "L": 1, "O": 1, "N": 1, "Q": 10, "P": 3, "S": 1, 
+    "R": 1, "U": 1, "T": 1, "W": 4, "V": 4, "Y": 4, 
+    "X": 8, "Z": 10
+}
+
+
+# result = random.choice(list(LETTER_POOL))
+# print(result)
+
+
+# result_2 = random.sample((list(LETTER_POOL)), 10)
+# print(result_2)
+
+
 def draw_letters():
-    pass
+    """
+    import random
+    ten_letters = [] 
+
+    * sytax for random module with dict: random.choice(list(LETTER_POOL))
+    drawing process:
+    for loop:
+        >assign randomly picked letter to variable selection
+        selection = random.choice(list(LETTER_POOL))
+        >if selection in LTTTER_POOL, LETTER_POOL[selection] -= 1
+            >if LETTER_POOL[selection] < 1, remove it(need deepcopy of dict)
+            > ten_letters.append
+    """
+    ten_letters = [] 
+    LETTER_POOL_02 = {}
+    for key, value in LETTER_POOL.items():
+        LETTER_POOL_02[key] = value
+    
+    
+    while len(ten_letters) < 10:
+        selection = random.choice(list(LETTER_POOL_02))
+        if selection in LETTER_POOL:
+            LETTER_POOL_02[selection] -= 1
+            if LETTER_POOL_02[selection] < 1:
+                LETTER_POOL_02.pop(selection)
+            ten_letters.append(selection)
+    return ten_letters
+
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    """
+    check if the word is an anagram of some or all of the given letters in the letter bank
+
+    created a freq dict of letter_bank: letter_bank_dict
+    created a freq dict of word: word_dict
+    for letter in word_dict:
+        if letter in letter_bank_dict:
+            if word_dict[letter] < letter_bank_dict[letter]  #compare the freq of same letter
+            return True
+
+    input.upper()
+    created a freq dict of letter_bank: letter_bank_dict
+    for letter in word:
+        if letter in letter_bank_dict:
+            letter_bank_dict[letter] -= 1
+            if letter_bank_dict[letter] < 1:
+                return False
+            else: 
+                return True        
+    """
+
+    # letter_bank = draw_letters()
+    letter_bank_dict = {}
+    word = word.upper()
+    for letter in letter_bank:
+        if letter in letter_bank_dict:
+            letter_bank_dict[letter] += 1
+        else:
+            letter_bank_dict[letter] = 1
+ 
+
+    for letter in word:
+        if letter in letter_bank_dict:
+            letter_bank_dict[letter] -= 1
+            if letter_bank_dict[letter] < 0:
+                return False
+        
+        else:
+            return False
+    return True
```

Comment:
👍  Nice work! You can omit some of the whitespace to make your code look neater.
```python
    letter_bank_dict = {}
    word = word.upper()
    for letter in letter_bank:
        if letter in letter_bank_dict:
            letter_bank_dict[letter] += 1
        else:
            letter_bank_dict[letter] = 1
 
    for letter in word:
        if letter in letter_bank_dict:
            letter_bank_dict[letter] -= 1
            if letter_bank_dict[letter] < 0:
                return False
        else:
            return False
    return True
```